cpp/log/tree_signer-inl.h

```diff
@@ -12,8 +12,10 @@
 #include <unordered_map>
 
 #include "log/database.h"
+#include "log/frontend.h"
```

Comment:
And I'm not too sure about this one either, but from a cursory look, it's not needed?